components/Loading/Loading.js

```diff
@@ -1,5 +1,5 @@
 import { Spinner } from "./Loading.styled";
 
-export default function Loading() {
-  return <Spinner></Spinner>;
+export default function Loading({ $small }) {
+  return <Spinner $small={$small}></Spinner>;
```

Comment:
It's a common practice to use the `$` sign only in the components where these properties are actually needed for the styled component. This keeps the upper component levels clean and avoids the propagation of specific style props through the whole component system, which facilitates reusability and maintenance.

**As already mentioned:**
If a component doesn’t contain any children, you can use the self-closing tag format for cleaner and more concise code.

```suggestion
export default function Loading({ small }) {
  return <Spinner $small={small} />
```